mlpipeline/mlpipeline/etl/pipeline.py

```
@@ -0,0 +1,234 @@
+
+        except S3Error as err:
+            print(f"Minio Error: {err}")
+        finally:
+            if response:
+                response.close()
+                response.release_conn()
+
+    def process_recipe(self, line: str) -> Optional[Exception]:
+        try:
+            recipe_data = Recipe.model_validate_json(line)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)